package.json

```diff
@@ -45,6 +45,7 @@
     "homepage": "https://github.com/hashicorp/vault-action#readme",
     "dependencies": {
         "got": "^11.8.5",
+        "hpagent": "^1.0.0",
```

Comment:
I see the latest version is 1.2.0 if we want to start with an up-to-date new dependency